core/modules/widgets/view.js

```diff
@@ -18,19 +18,95 @@ var ViewWidget = function(parseTreeNode,options) {
 	this.initialise(parseTreeNode,options);
 };
 
+var ViewHandler = function(parseTreeNode,options) {
+	this.initialise(parseTreeNode,options);
```

Comment:
`this.initialise(parseTreeNode,options);` handles the initialisation of the options-object. So here it is OK to use it that way

Reply:
This is now changed